docs/markdown/snippets/vs_module_defs_customtargetindex.md

```diff
@@ -0,0 +1,8 @@
+## vs_module_defs keyword now supports indexes of custom_target
+
+This means you can do something like:
+```meson
+defs = custom_target('generate_module_defs', ...)
+shared_library('lib1', vs_module_defs : defs[0])
+shared_library('lib2', vs_module_defs : defs[2])
```

Comment:
Apart from that it's strange to take CustomTarget but not CustomTargetIndex, it's likely niche that there's a real reason for it. The most realistic one I can think of is that you're generating code and the vs_module_defs together from a representative source of some kind, but you want to use a module defs file instead of the pre-processor:
```meson
ct = custom_target(
  'from_xml',
  input : 'functions.xml',
  output : ['functions.hpp', 'functions.cpp', 'functions.def'],
  command : ['xml-to-code', '@INPUT@', '@OUTPUT@'],
)
both_library(
  'generated',
  ct[0], ct[1],
  vs_module_defs : ct[2],
)
```